mne_bids/read.py

```diff
@@ -148,8 +149,14 @@ def _read_events(events_data, event_id, raw, verbose=None):
     )
     all_dur = raw.annotations.duration
     if all_events.size == 0:
-        warn('No events found or provided. Please add annotations to the raw '
-             'data, or provide the events_data and event_id parameters.')
+        #Confirm from filename dataset is not likely resting state
+        fileparts = op.split(raw.filenames[0])
+        if not re.search('rest',fileparts[-1]):
+            warn('No events found or provided. Please add annotations to the raw '
+                    'data, or provide the events_data and event_id parameters.')
```

Comment:
```suggestion
            warn('No events found or provided. Please add annotations to the raw '
                 'data, or provide the events_data and event_id parameters.')
```

Reply:
but @hoechenberger I would actually not warn at all. I would just log things when verbose=True

Reply:
I think the problem may be that when no events are found, no events.tsv is written, and then the data may be invalid BIDS ... users may not realize their error if they have verbose = False. 

So I think this should be logged or warned about even when verbose=False.

However for the case of resting data (this PR), I am +1 to not log/warn